source/languages/en/dataplatform/upgrading.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Upgrading
+project: dataplatform
+version: 1.0.0+
+document: guide
+toc: false
+index: true
+audience: beginner
+---
+
+#Upgrading from Basho Data Platform Beta to 1.0.0
+
+>**NOTE: 
```

Comment:
<div class="note">
<div class="title">Note:</div>
This page only applies to people previously using the beta releases of Basho Data Platform.
</div>